docs/releases/1.10.txt

```diff
@@ -161,6 +161,9 @@ Management Commands
   command exit with a non-zero status when model changes without migrations are
   detected.
 
+* The Web server launched by the :djadmin:`runserver` command claims HTTP 1.1
```

Comment:
So you can already use the server with HTTP 1.1 and this change just affects clients who respect what the web server reports it supports?